spec/requests/custom_button_sets_spec.rb

```diff
@@ -199,4 +199,19 @@
       expect(response.parsed_body).to include(expected)
     end
   end
+
+  describe 'OPTIONS /api/custom_button_sets' do
+    it 'returns custom_button_sets for Generic Object Definitions and Service Templates' do
+      options(api_custom_button_sets_url)
+
+      expected_data = {
+        'custom_button_sets' => {
+          'generic_object_definitions' => CustomButtonSet.find_all_by_class_name('GenericObjectDefinition'),
+          'service_templates'          => CustomButtonSet.find_all_by_class_name('ServiceTemplate')
```

Comment:
This is not right to create a test that is the exact same code at the code itself.  You're not really testing anything.  cc @chrisarcand Thoughts?

Reply:
Agree that the test here is not ideal - and mocking the expected result is also not an option.
So yes, ideas are welcome.

Reply:
On second thoughts, I think we could write a proper test here.

Although I need to know if we are going ahead with this PR, before I rectify the spec.

Reply:
I know this PR is scrapped but just to answer the question for future knowledge...

Yes, you should be creating those custom button sets ahead of time with helpers/variables and referring to those directly in your test, ie:

```ruby
expected_data = {
  'custom_button_sets => {
    'generic_custom_object_definitions' => cb_set # You created cb_set earlier, now refer to it
# ......
```